color_test.go

```diff
@@ -3,38 +3,36 @@ package lipgloss
 import (
 	"image/color"
 	"testing"
-
-	"github.com/muesli/termenv"
 )
 
 func TestSetColorProfile(t *testing.T) {
-	r := renderer
+	r := DefaultRenderer()
 	input := "hello"
 
 	tt := []struct {
 		name     string
-		profile  termenv.Profile
+		profile  Profile
 		expected string
 	}{
 		{
 			"ascii",
-			termenv.Ascii,
+			Ascii,
 			"hello",
 		},
 		{
 			"ansi",
-			termenv.ANSI,
-			"\x1b[94mhello\x1b[0m",
+			ANSI,
+			"\x1b[94mhello\x1b[m",
```

Comment:
should these still end with `[0m`?

Reply:
Short answer: they're the same, `[m` is shorter and consumes one less byte
Long answer: SGR style sequences default value is `0` meaning terminals treat a missing parameter for SGR i.e. `CSI p...p m` sequences as zero. Thus, `CSI m` is equal to `CSI 0 m`

Reply:
oh, TIL!

thanks for explaining!